downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,218 @@
+## Overview of Ballerina Swan Lake Beta1
 
-### Updating Ballerina
+<em>This is the Beta1 release in a series of planned Beta releases leading up to the Ballerina Swan Lake GA release.</em> 
 
-You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+It introduces the new language features planned for the Swan Lake GA release and includes improvements and bug fixes done to the compiler, runtime, standard library, and developer tooling after the Swan Lake Alpha5 release.
 
-#### For existing users
+## Updating Ballerina
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using Ballerina, you can use the [Update Tool](/learn/tooling-guide/cli-tools/update-tool/) to directly update to Ballerina Swan Lake Beta1 as follows. 
 
-> $ ballerina dist update
+To do this, first, execute the command below to get the update tool updated to its latest version. 
 
-However, you need to use the following commands instead of the above if you have installed:
+> `bal update`
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Beta1.
 
-#### For new users
+> `bal dist pull beta1`
 
-If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+## Installing Ballerina
 
-### Highlights
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
 
-### What is new in Ballerina Swan Lake Preview <VERSION>
+## Language Updates
 
-#### Language
--
+### Improvements
 
-#### Runtime
--
+- Worker deadlock detection has been improved to include the `wait` action.
+- The `self` variable of an isolated object no longer needs to be accessed within a `lock` statement unless it is used to access a field that is either not `final` or is of a type that is not a subtype of `readonly` or `isolated object {}`.
+- The `lang.xml:strip()` function does not mutate an XML value. Therefore, using the `lang.xml:strip()` function on immutable XML values is allowed.
 
-#### Standard Library
-- 
+### Breaking Changes
 
-#### Developer Tools
+- The Taint analyzer is disabled and the `--taint-check` option is removed. The annotations `@tainted` and `@untainted` were left intact as placeholders for backward compatibility. Any script that used to pass the `--taint-check` option to the `bal run` or `bal build` commands will fail.
```

Comment:
Shall we make this the last one in the list? IMO, the others are more significant since this was not enabled by default.